### PR TITLE
Load ads directly instead of pulling from iframe

### PR DESCRIPTION
**Pull request by msub2**
This PR changes the behavior of the SDK to pull the wrapper in to the embedding page directly instead of going through the cross-origin frame, which was impacting viewability metrics. It also no longer initializes prebid resources if the ad unit ID provided is not a valid v4 uuid, along with a slight change to the A-Frame tests to account for varying image src.

### CODE REVIEW

File: tests/aframe.test.mjs [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import { test, expect } from '@playwright/test';
 
+function srcEvaluate(node) {
```
Comment (limbofeather) [inline tests/aframe.test.mjs:3]: 🙏🏼

**Comment by limbofeather** [comment]
Would be good to leave a note on how to update the WLE test projects in the SDK README.

File: utils/networking.js [hunk 2]
```diff
@@ -17,57 +17,82 @@ const AD_REFRESH_INTERVAL = 15000;
...
   // Load gifler script in case gif creative is served
-  const script = document.createElement('script');
-  script.src = 'https://cdn.jsdelivr.net/npm/gifler@0.1.0/gifler.min.js';
-  document.head.appendChild(script);
+  const gifscript = document.createElement('script');
+  gifscript.src = 'https://cdn.jsdelivr.net/npm/gifler@0.1.0/gifler.min.js';
+  document.head.appendChild(gifscript);
+
+  // Select baseDivId based on format, defaulting to the one for medium rectangle
+  if (format == 'billboard') {
```
Comment (limbofeather) [inline utils/networking.js]: Could all of this information on each format's attributes be in `utils/formats.js`?

**Review by limbofeather — APPROVED** [review]
Small nit but not blocker.

**Comment by msub2** [comment]
> Would be good to leave a note on how to update the WLE test projects in the SDK README.

I've got an existing issue at #206 which would remove the need to manually build the test project, just haven't gotten around to figuring out the workflow yet

File: utils/networking.js (hunk 2 above)
Comment (msub2) [reply utils/networking.js]: It probably could yeah, can do in a small followup if this deploys without issue

**Pull request merged**